tests/run-end-to-end.sh

```diff
@@ -55,13 +52,25 @@ flowctl temp-data-plane \
     --unix-sockets \
     &
 DATA_PLANE_PID=$!
+
+# `flowctl temp-data-plane` always uses ./builds/ of --tempdir as its --flow.builds-root.
+# See cmd-temp-data-plane.go.
+export BUILDS_ROOT=${TESTDIR}/builds
```

Comment:
This can be removed now, right?

Reply:
It's still used, but only locally within the script as part of the `mv` below.